jormungandr-lib/src/interfaces/vote.rs

```diff
@@ -471,6 +471,23 @@ mod serde_base64_bytes {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct TallyDecryptShare(#[serde(with = "serde_base64_bytes")] Vec<u8>);
+
+impl TryFrom<TallyDecryptShare> for chain_vote::TallyDecryptShare {
+    // TODO: Maybe change this error to a custom type?
+    type Error = std::io::Error;
```

Comment:
... then you can use `certificate::Error` for the command implementation.

This impl looks atypical for what the rest of jormungandr-lib does, let's not add it here until it's clear that we also need the serde impls for REST.